views/index.pug

```diff
@@ -1,21 +1,68 @@
 html
-  include partials/head
+head
+    link(rel="stylesheet" href="/css/styles.css")
 
-  body
-    h1= schittscreek.name
+    body
+     h1.title Welcome to Schitt's Creek!
+      h2 Select a Method:
+      li.methodsInfo 
+        p 
+          ul.methods GET/allCharacters
+        div#getAll
+            p Method: GET 
+            p Route: http://localhost:4053/characters/allCharacters 
+            p Headers:   
+            p Description: Returns a list of all of the characters in Schitt's Creek.
+      li.methodsInfo 
+        p 
+          ul.methods GET/characterByName
+        div 
+            p Method: GET 
+            p Route: http://localhost:4053/characters/characterByName 
+            p Headers:   
+            p Description: Returns the character with the name provided in the route.  
+      li.methodsInfo 
+        p
+          ul.methods GET/characterByQuote
+        div 
+            p Method: GET 
+            p Route: http://localhost:4053/characters/characterByQuote
```

Comment:
Sending long data like a quote up in a URL for a GET will definitely be tricky from an encoding/decoding persepctive.